index.rst

```diff
@@ -6,19 +6,14 @@ Solidity
     :alt: Solidity logo
     :align: center
 
-Solidity is a contract-oriented, high-level language whose syntax is similar to that of JavaScript
-and it is designed to target the Ethereum Virtual Machine (EVM).
+Solidity es un lenguaje de alto nivel orientado a escribir contratos. Su sintaxis es similar a la de JavaScript y está diseñado para enfocarse a la Máquina Virtual de Etehreum (EVM).
```

Comment:
It could be "orientado a contratos" but I was afraid of confusing people with other meanings of the word "contract". I'm not satisfied with the translation of this intro to be honest, I'll most certainly do it again.
Thanks for your comment Chris